src/main/java/com/gmail/yuramitryahin/service/ShoppingCartServiceImpl.java

```diff
@@ -0,0 +1,47 @@
+package com.gmail.yuramitryahin.service;
+
+import com.gmail.yuramitryahin.dao.ShoppingCartDao;
+import com.gmail.yuramitryahin.dao.TicketDao;
+import com.gmail.yuramitryahin.lib.Inject;
+import com.gmail.yuramitryahin.lib.Service;
+import com.gmail.yuramitryahin.model.MovieSession;
+import com.gmail.yuramitryahin.model.ShoppingCart;
+import com.gmail.yuramitryahin.model.Ticket;
+import com.gmail.yuramitryahin.model.User;
+
+@Service
+public class ShoppingCartServiceImpl implements ShoppingCartService {
+    @Inject
+    private ShoppingCartDao shoppingCartDao;
+    @Inject
+    private TicketDao ticketDao;
+
+    @Override
+    public void addSession(MovieSession movieSession, User user) {
+        Ticket ticket = new Ticket();
+        ticket.setMovieSession(movieSession);
+        ticket.setUser(user);
+        ticketDao.add(ticket);
```

Comment:
if there is no shopping cart for the user - we still save ticket to the db, is it ok?

Reply:
not ok, fixed